attribute_set_completeness/components/attribute_set_owner_event_listener.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.addons.component.core import AbstractComponent
+from odoo.addons.component_event import skip_if
+
+
+class AttributeSetOwnerEventListener(AbstractComponent):
```

Comment:
@Cedric-Pigeon in this case a simple override of write would have been enough IMO :)